src/vs/workbench/parts/files/electron-browser/fileActions.ts

```diff
@@ -725,6 +722,42 @@ class BaseDeleteFileAction extends BaseFileAction {
 			});
 		});
 	}
+
+	private getMoveToTrashMessage(distinctElements): string {
+		let message: string = '';
+		if (distinctElements.length > 1) {
+			if (distinctElements[0].isDirectory) {
+				message = getConfirmMessage(nls.localize('confirmMoveTrashMessageMultipleDirectories', "Are you sure you want to delete the following {0} directories and its contents?", distinctElements.length), distinctElements.map(e => e.resource));
+			} else {
+				message = getConfirmMessage(nls.localize('confirmMoveTrashMessageMultiple', "Are you sure you want to delete the following {0} files?", distinctElements.length), distinctElements.map(e => e.resource));
+			}
+		} else {
+			if (distinctElements[0].isDirectory) {
+				message = nls.localize('confirmMoveTrashMessageFolder', "Are you sure you want to delete '{0}' and its contents?", distinctElements[0].name);
+			} else {
+				message = nls.localize('confirmMoveTrashMessageFile', "Are you sure you want to delete '{0}'?", distinctElements[0].name);
+			}
+		}
+		return message;
+	}
+
+	private getDeleteMessage(distinctElements): string {
```

Comment:
@pradeepmurugesan the `distinctElements` is missing a type annotation: `distinctElements: ExplorerItem[]`

Reply:
fixed